Tasks/app-store-release/app-store-release.ts

```diff
@@ -43,6 +43,27 @@ function findIpa(ipaPath: string): string {
     return paths[0];
 }
 
+async function isFastlaneInstalled(): Promise<boolean> {
+    tl.debug('Checking for previous fastlane version first...');
+    let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
+    gemRunner.arg(['list', 'fastlane']);
+    gemRunner.arg(['-i']);
+    let fastlaneListOutput: string;
+
+    gemRunner.on('stdout', (data) => {
```

Comment:
I would just use `execSync` here.  That will just give you an object with all the stdout, stderr, etc.

Reply:
+1